### DISCUSSION TITLE
【第6回】2024年04月21日

### DISCUSSION THREAD

**Discussion by kagomen** [first-contributions-ja/first-contributions-ja.github.io#173] [【メンテナー必読】週報]
## 目次
> 目次はトピックを立てた人が書き加えてください

- [画像化する絵文字のフォントについて](https://github.com/first-contributions-ja/first-contributions-ja.github.io/discussions/173#discussioncomment-9177441)
  - 結論: Appleアイコンの次に見慣れているTwemoji
- [デザイン案の持ち寄り](https://github.com/first-contributions-ja/first-contributions-ja.github.io/discussions/173#discussioncomment-9178620)
  - 結論:
    - 仮のデザイン案をもとに調査を進めていく。調査後、改善・修正箇所が判明したら（必要であれば）最終的なデザインカンプを用意し、その後実装に移る。 
    - デザイン案は[結論](https://github.com/first-contributions-ja/first-contributions-ja.github.io/discussions/173#discussioncomment-9178620)と[Figma](https://www.figma.com/file/xTVaHLu2OeeIBS8nGeDAvC/%E8%80%83%E3%81%88%E3%81%A6%E3%81%84%E3%82%8B%E3%81%93%E3%81%A8%E3%81%AE%E3%81%BE%E3%81%A8%E3%82%81?type=whiteboard&node-id=0-1&t=U87dwKfpryYub03d-0)を参照

- どのようにユーザーにLGTM画像を共有するか？
  - DL用ページを生成する
  - DLページがあるとツイッターでシェアする際にOGP画像も生成できる！
  - ※静的サイトだと、クエリ文で画像を生成できない

- Contributions.jsonに記入する内容はどうするか？
  - 挨拶を含めるか？ => 含めない
    - 理由:
      - スマホ版とPC版でデザインを別にする場合、実装期限（4月末、遅くても5月）に間に合わないので、スマホベースで考える
      - スマホ版はメッセージを表示させる領域がない
  - 絵文字の個数は？ => 未定
    - フォントの取得方法について解明したら、デザインを確定させたのち、決める

```json
    {
    "name": "あなたのハンドルネーム",
    "github": "あなたのGitHubアカウントのURL",
    "favoriteColor": "あなたの好きなカラーコード",
    "favoriteEmoji": "あなたの好きな絵文字"
    }
```

- コンセプトについて
  - [Wiki](https://github.com/first-contributions-ja/first-contributions-ja.github.io/wiki/%E3%81%93%E3%81%AE%E3%83%97%E3%83%AD%E3%82%B8%E3%82%A7%E3%82%AF%E3%83%88%E3%81%AE%E7%9B%AE%E7%9A%84%E3%82%84%E3%82%B3%E3%83%B3%E3%82%BB%E3%83%97%E3%83%88%E3%81%AB%E3%81%A4%E3%81%84%E3%81%A6)に書きました
  - なにか修正点や疑問点などありましたら直接修正していただくか、discordやdiscussionsで声をかけてください！
 
## 次の一週間でやること
- ヒーローセクションの絵文字の表示の仕方について、調査
- DLページのデザイン案を簡単に考える
- DLしたフォントを画像生成で使用できるのか、調査
- アバウトセクションの文章をつめていく
  - 画像や絵文字を使って明るくしたい！

***
以下フォーマット
```
## トピック: <ここにタイトル>
<ここに概要>

## 結論
<ここに結論>

## 関連 Issues、Discussions

```

**Comment by kazzyfrog** [comment]
## トピック:  画像化する絵文字のフォントについて
絵文字を画像化する際に、appleの絵文字は著作権的に使えない。
なので、別の絵文字フォントを使う必要がある。

下記のライブラリを使おうと思っているので、それのデモサイトで絵文字のフォントがいくつか紹介されている：
https://og-playground.vercel.app/

figmaにも、絵文字のデモを貼ってます！！

## 結論
Appleアイコンの次に見慣れているTwemojiを使用する！

## 関連 Issues、Discussions

**Comment by kagomen** [comment]
## トピック: デザイン案の持ち寄り
途中でもOK！
確認し、何かデザインに関して進展があれば結論に記入する

## 結論
仮のデザイン案をもとに調査を進めていく。
調査後、改善・修正箇所が判明したら最終的なデザインカンプを用意し、その後実装に移る。

### 仮デザイン案
<img width="375" alt="FCJv2" src="https://github.com/first-contributions-ja/first-contributions-ja.github.io/assets/154225199/f1a8e501-be61-46e2-a2d3-d52cd5e47753">

![LGTM画像](https://github.com/first-contributions-ja/first-contributions-ja.github.io/assets/154225199/c3cea3ce-78b8-4815-84f6-7840bd3a21a8)


## 関連 Issues、Discussions
#116
